modules/recommended-etcd-practices.adhoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/recommended-etcd-practices.adoc
```

Comment:
Do you want this as part of the scalability_and_performance/recommended-host-practices.adoc file or as a new, standalone topic? If it's a new topic, then we need to add it to the _topic_map.yml file

Reply:
In 3.11, etcd best practices were part of recommended-host-practices. So I think we can have the etcd guidelines at the same place in 4.1 as well.

Reply:
Agreed, added this to the recommended-host-practices.adhoc file.